docs/baking.md

```diff
@@ -0,0 +1,185 @@
+file, you should reload systemctl daemon via running `sudo systemctl daemon-reload`.
+
+### Bootstrapping the node
+
+In order to run a baker locally, you'll need a fully-synced local `tezos-node`.
+
+The fastest way to bootstrap the node is to import a snapshot.
+Snapshots can be downloaded from https://snapshots.tulip.tools/#/ .
+
+All commands within the service are run under the `tezos` user.
+
+Download the snapshost for desired network and with the desired history mode
+(you can read more about `tezos-node` history modes [here](https://tezos.gitlab.io/user/history_modes.html#history-modes)).
+
+Snapshot import requires that the `tezos-node` data directory doesn't contain the `context` and `store` folders.
+Remove them before the snapshot import.
+
+In order to import the snapshot run the following command:
+```
+sudo -u tezos-node snapshot import --data-dir /var/lib/tezos/node-<network>/ <path to the snapshot file>
+```
+
+In case you're getting an error similar to
+```
+tezos-node: Error:
+              Invalid block BLjutMj47caB
+                Failed to validate the economic-protocol content of the block: Error:
+                                                                                Invalid signature for block BLjutMj47caB. Expected: tz1Na5QB98cDA.
+```
+
+You should init/update config in the data directory to match the desired network
+before importing the snapshot. To do that run one of the following commands
+depending on whether the config in the given data directory was initialized previously:
+```
+sudo -u tezos tezos-node config init --data-dir /var/lib/tezos/node-<network> --network <network>
+#or
+sudo -u tezos tezos-node config update --data-dir /var/lib/tezos/node-<network> --network <network>
+```
+
+### Starting the node
+
+After the snapshot import, you can finally start the node by running
+```
+sudo systemctl start tezos-node-<network>.service
+```
+
+Note that even after the snapshot import the node can still be out of sync, it may require
+some additional time to completely bootstrap.
+
+To stop the node run:
+```
+sudo systemctl stop tezos-node-<network>.service
+```
+
+You can check the node logs via `journalctl`:
+```
+journalctl -u tezos-node-<network>.service
+```
+
+## Setting up baker and endorser daemons
+
+Once you have fully synced your local `tezos-node`, you can start the baker and endorser daemons.
+
+### Setting up daemons data directory
+
+Data directories for baker and endorser daemons are defined in the
+`/etc/default/tezos-baker-<proto>` and `/etc/default/tezos-endorser-<proto>`. Make
+sure to point them at the same directory to have the same set of known keys, e.g.
+point both `DATA_DIR`s to `/var/lib/tezos/baker`. You should also
+update `NODE_DATA_DIR` in the `/etc/default/tezos-baker-<proto>` to point at the desired
+node data directory, e.g. `/var/lib/tezos/node-<network>`.
+
+Create the aformentioned daemons data directory:
+```
+sudo -u tezos mkdir -p /var/lib/tezos/baker
+```
```

Comment:
why does this require user action at all? the defaults should be correct out of the box, `/var/lib/tezos/baker` should be part of `.deb` package

Reply:
Agree, created #133